MovieMVP/ListMovieModule/View/ListMoviesViewController.swift

```diff
@@ -174,72 +175,52 @@ final class ListMoviesViewController: UIViewController {
         }
     }
 
-    private func fetchData(categoryMovies: String) {
-        let urlString =
-            "\(Constants.themoviedbQueryText)\(categoryMovies)\(Constants.apiKeyQueryText)" +
-            "\(Constants.languageQueryText)\(Constants.pageQueryText)\(Constants.pageQueryText)"
-        guard let url = URL(string: urlString) else { return }
-        let task = URLSession.shared.dataTask(with: url) { data, _, error in
-            if error == nil {
-                self.decodeData(data: data)
-                DispatchQueue.main.async {
-                    self.mainActivityIndicatorView.stopAnimating()
-                    self.mainActivityIndicatorView.isHidden = true
-                    self.listMoviesTableView.reloadData()
-                }
-            } else {
-                guard let safeError = error else { return }
-                print(safeError)
-            }
-        }
-        task.resume()
-    }
-
-    private func decodeData(data: Data?) {
-        let decoder = JSONDecoder()
-        guard let safeData = data else { return }
-        do {
-            let decodedData = try decoder.decode(MovieList.self, from: safeData)
-            movies = decodedData.movies
-        } catch {
-            print(error)
-        }
-    }
-
     private func createPopularButtonConstraint() {
         popularButton.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
-            popularButton.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor, constant: 5),
+            popularButton.topAnchor.constraint(
+                equalTo: view.layoutMarginsGuide.topAnchor,
+                constant: Constants.fiveNumber
+            ),
             popularButton.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor),
             popularButton.widthAnchor.constraint(equalTo: topRatedButton.widthAnchor),
-            popularButton.heightAnchor.constraint(equalToConstant: 40)
+            popularButton.heightAnchor.constraint(equalToConstant: Constants.fortyNumber)
```

Comment:
```suggestion
            popularButton.heightAnchor.constraint(equalToConstant: Constants.popularButtonHeight)
```
100% неверный нэйминг тут и до и после. Смысл в том, что ты высоту, длину, массу,  текст КАКОЙ-ТО СУЩНОСТИ задаешь в const и при необходимости там ЛЕГКО ЕЕ НАХОДИШЬ изменяешь 40 на 45 и что дальше? fortyNumber переименовать в fifty? тут константа height, она так и должна называться height чего-то например nameLabel или nextButton

Reply:
Всегда так делал замечаний со стороны менторов не было у меня тут константы используются только как числа не более, иначе у каждой цифры будет свой константа, которых будет много и каждая будет применяться единожды, не знаю есть ли смысл тогда константы было выносить если применяется она 1 раз